src/auth/guards/jwt-auth.guard.ts

```
@@ -1,5 +1,31 @@
+    }
+
+    return super.canActivate(context);
+  }
+
+  handleRequest(err: any, user: any) {
+    if (err || !user) {
+      throw err || new UnauthorizedException('Invalid or expired token');
+    }
+    return user;
+  }
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)